appwrite/appwrite
# feat: x domain cookie

## What does this PR do?

Adds an additional env var `_APP_CONSOLE_ROOT_SESSION`

When console sessions will be stored on the root of the host domain.

e.g. if the host domain is cloud.appwrite.io, and `_APP_CONSOLE_ROOT_SESSION=enabled` , console sessions with have .appwrite.io cookie domain.

## Test Plan

Manual testing: 

Create a console account on https://x-domain-cookie-test.appwrite.org, where `_APP_CONSOLE_ROOT_SESSION=enabled` and verify the cookie domain is .appwrite.org.

Create a non console / project account on https://x-domain-cookie-test.appwrite.org and verify the cookie domain is .x-domain-cookie-test.appwrite.org

## Checklist
